src/components/number-picker/number-picker.component.html

```diff
@@ -6,6 +6,7 @@
     (ngModelChange)="_emitValueChange($event)"
     [min]="min"
     [max]="max"
+    [placeholder]="placeholder || ''"
```

Comment:
This can be `[placeholder]="placeholder"`, if it's undefined then it will not appear in the DOM.

Reply:
Done the changes